tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
Yes, I think that conceptually makes sense, although I wouldn't literally automatically set `editOptions.place` since that would lead to unexpected results if the app called `get("editOptions")`, or if it set `editOptions` programatically after initialization.

It looks like there's already code that gets the effective edit options by combining `constraints` and `editOptions` together, either by using `lang.delegate()` or `lang.mixin()`.   So I think it's just a question of enhancing that code.